src/core/interfaces/configuration.ts

```diff
@@ -786,10 +786,19 @@ export const setDefaults = (configuration: UserConfiguration): ExtendedConfigura
     secretKey: configurationCopy.secretKey,
   };
 
-  if (configurationCopy.presenceTimeout !== undefined && configurationCopy.presenceTimeout < PRESENCE_TIMEOUT_MINIMUM) {
-    configurationCopy.presenceTimeout = PRESENCE_TIMEOUT_MINIMUM;
-    // eslint-disable-next-line no-console
-    console.log('WARNING: Presence timeout is less than the minimum. Using minimum value: ', PRESENCE_TIMEOUT_MINIMUM);
+  if (configurationCopy.presenceTimeout !== undefined) {
+    if (configurationCopy.presenceTimeout > PRESENCE_TIMEOUT_MAXIMUM) {
+      configurationCopy.presenceTimeout = PRESENCE_TIMEOUT_MAXIMUM;
+      // eslint-disable-next-line no-console
+      console.warn(
+        'WARNING: Presence timeout is larger than the maximum. Using maximum value: ',
+        PRESENCE_TIMEOUT_MAXIMUM,
+      );
+    } else if (configurationCopy.presenceTimeout <= 0) {
+      // eslint-disable-next-line no-console
+      console.warn('WARNING: Presence timeout is larger than zero.');
```

Comment:
nitpick - Shouldn't this be like "Presence timeout should be larger than zero" ? when value is negative or 0